src/client/components/ImageMenu.js

```diff
@@ -103,28 +105,43 @@ const ImageMenu = ({
     boxes[ethereumAddress].publicProfile[imageTag][0].contentUrl['/']
 
   return (
-    <ImageMenuStyled
-      top={top}
-      right={right}
-      dragging={dragState.dragging}
-      {...getRootProps({
-        className: 'dropzone',
-        isDragActive,
-        isDragAccept,
-        isDragReject,
-        imageCid,
-      })}
-    >
-      {' '}
-      <div>Update {imageTitle} photo</div>
-      <input {...getInputProps({ disabled: false })} />
-      <div onClick={open}>Upload new image</div>
-      {imageExists && (
-        <div onClick={() => dispatchModal(removeItem(imageCid, imageTag))}>
-          Delete
+    <div tabIndex="-1" onBlur={() => setActive(false)}>
+      <ImageMenuStyled
+        top={top}
+        right={right}
+        dragging={dragState.dragging}
+        active={active}
+        {...getRootProps({
+          className: 'dropzone',
+          isDragActive,
+          isDragAccept,
+          isDragReject,
+          imageCid,
+        })}
+      >
+        <div css="padding: 8px 8px 6px 8px" onClick={() => setActive(!active)}>
+          <IconCamera width="16px" height="16px" css="margin-right: 8px" />
+          Update {imageTitle} photo
         </div>
-      )}
-    </ImageMenuStyled>
+
+        {active && (
+          <React.Fragment>
```

Comment:
While I'm thinking about accessibility, another interaction I anticipate being strange for users of screen readers:

Once "Update Cover photo" is a button, and therefore findable by screen readers, here's what will happen:

* I click "Update Cover photo"
* ...

Nothing. I don't know that new stuff appeared on the screen. I would have to start reading through the page again or something to discover that there are two new buttons available.

I think we can fix that by using an `aria-live` region. Here's how we implement it:

```jsx
<button onClick={() => setActive(true)} aria-controls={`${imageTag}-photo-actions`}>
  Update {imageTitle} photo
</button>
<div role="region" aria-live="polite" id={`${imageTag}-photo-actions`}>
  {active && (
    <React.Fragment>...</React.Fragment>
  )}
</div>
```

Here's how it works:

* on initial page load, the screen reader makes note of this "aria-live" element
* when the content in this region changes, the screen reader breaks the flow of whatever it's doing and reads the content

Learn more here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions